osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -2501,6 +2501,70 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/xcn.svg"
       },
       "coingecko_id": "chain-2"
+    },
+	  {
+      "description": "Wrapped GLMR on Axelar",
+      "type_asset": "erc20",
+      "denom_units": [
+        {
+          "denom": "ibc/1E26DB0E5122AED464D98462BD384FCCB595732A66B3970AE6CE0B58BAE0FC49",
+          "exponent": 0,
+          "aliases": [
+            "wglmr-wei",
+            "0xacc15dc74880c9944775448304b263d191c6077f"
+          ]
+        },
+        {
+          "denom": "axlwglmr",
+          "exponent": 18
+        }
+      ],
+      "base": "ibc/1E26DB0E5122AED464D98462BD384FCCB595732A66B3970AE6CE0B58BAE0FC49",
+      "name": "Wrapped GLMR",
+      "display": "axlwglmr",
+      "symbol": "WGLMR.axl",
+      "ibc": {
+        "source_channel": "channel-3",
+        "dst_channel": "channel-208",
+        "source_denom": "wglmr-wei"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/moonbeam/images/glmr.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/moonbeam/images/glmr.svg"
+      },
+      "coingecko_id": "wrapped-moonbeam"
+    },
+    {
+      "description": "Wrapped Polkadot on Axelar",
+      "type_asset": "erc20",
+      "denom_units": [
+        {
+          "denom": "ibc/3FF92D26B407FD61AE95D975712A7C319CDE28DE4D80BDC9978D935932B991D7",
+          "exponent": 0,
+          "aliases": [
+            "dot-planck",
+            "0xffffffff1fcacbd218edc0eba20fc2308c778080"
+          ]
+        },
+        {
+          "denom": "axlxcdot",
+          "exponent": 10
+        }
+      ],
+      "base": "ibc/3FF92D26B407FD61AE95D975712A7C319CDE28DE4D80BDC9978D935932B991D7",
+      "name": "Wrapped Polkadot",
+      "display": "axlxcdot",
+      "symbol": "xcDOT.axl",
```

Comment:
Let's remove the "xc".  I think it's too confusing

Reply:
although Axelar recognizes that Moonbean calls it xcDOT, Axlear's asset metadata doesn't include the xc- prefix, so I've removed it from the denom as well.